src/m_cheat.c

```diff
@@ -660,8 +660,8 @@ static void cheat_clev(char *buf)
 // killough 2/7/98: simplified using dprintf and made output more user-friendly
 static void cheat_mypos()
 {
-  plyr->powers[pw_renderstats] = 0;
-  if (!(plyr->powers[pw_mapcoords] ^= 1))
+  plyr->cheats &= ~CF_RENDERSTATS;
+  if (!((plyr->cheats ^= CF_MAPCOORDS) & CF_MAPCOORDS))
```

Comment:
Thanks for the review! I'd like to deobfuscate this part a bit before merging, though.